snap/README.md

```diff
@@ -0,0 +1,43 @@
+# Kata Containers Snap
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu Bionic*
+
+```
+$ sudo apt-get install -y snap
+$ git clone https://github.com/kata-containers/packaging
+```
+
+## Build snap image
+
+Run next command at the root directory of the packaging repository.
+
+```
+$ snapcraft -d
+```
+
+## Install snap (developer)
+
+```
+$ sudo snap install --classic --dangerous kata-containers_[VERSION]_[ARCH].snap
```

Comment:
Could you add a note explaining what `--classic` and particularly what `--dangerous` does? :)